radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/messaging/InboundMessage.java

```diff
@@ -124,13 +82,15 @@ public boolean equals(Object obj) {
 		}
 		if (obj instanceof InboundMessage) {
```

Comment:
you can use pattern matching in instanceof and remove next line:

`if (obj instanceof InboundMessage other) {`

Reply:
nice :+1: